Doc/faq/design.rst

```diff
@@ -290,6 +292,9 @@ It's suggested that you use a prefix for the method names, such as ``visit_`` in
 this example.  Without such a prefix, if values are coming from an untrusted
 source, an attacker would be able to call any method on your object.
 
+Imitating switch with fallthrough, as with Fortran's computed GOTO
+and C's switch-case-default, is possible, much harder, and less needed.
```

Comment:
I probably wouldn't mention Fortran, at least not as the first example.

Reply:
I agree.  I got that example from the Wikipedia discussion, which likely included it to show that C was not unique.  But that point is not needed here, and indeed though I programmed Fortran before C, I would not have thought  of it.  Anyone who would likely knows of C also.